.travis.yml

```diff
@@ -1,15 +1,7 @@
 language: python
-python: 2.7
```

Comment:
`-python 2.7` isn't required as by default travis will expose all available python interpreters.

The `matrix` part shouldn't be needed anymore, as travis should provide the 3.5 interpreter. (this needs to be verified)